File: intro/code/methods/FileReading.java
package methods;

import java.io.File;
import java.util.Scanner;

public class FileReading {

    public static void main(String[] args) {
        File f = new File("pg100.txt");
        try {
            Scanner reader = new Scanner(f);
            int count =0;
            while(reader.hasNextLine()) {
                String line = reader.nextLine();

                count++;
            }
            System.out.println(count);
        } catch (Exception e){
            System.out.println("uhhhh.....");
            e.printStackTrace();
        }
    }
}

File: itp/fileReading.py
data = open("testFile.txt", 'r')

for line in data:
    print(line.strip())



"""
try{
    Scanner  scanner = new Scanner(new File("testFile.txt"));
    while(scanner.hasNextLine()) {
        String line = scanner.nextLine();
        System.out.println(line);
    }
catch(Exception e) {
    e.printStackTrace();
}

"""
